# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/xpcshell.ini
# Commit: 8e4f927ad236
# Full Hash: 8e4f927ad236e3f39665ae7aca84783989a9aaa2
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2020-09-15 15:47:19
# Regressor Bug: 1645108
# File Overlap Count: 2
# Description:
#   Bug 1645108 - Parse additional section of TRR response r=dragana,necko-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D87088
# ==============================================================================

diff -r 8342bb37c478 -r 8e4f927ad236 netwerk/test/unit/xpcshell.ini
--- a/netwerk/test/unit/xpcshell.ini	Tue Sep 15 10:32:29 2020 +0000
+++ b/netwerk/test/unit/xpcshell.ini	Tue Sep 15 10:38:27 2020 +0000
@@ -451,5 +451,8 @@
 skip-if = asan || tsan || os == 'win' || os =='android'
 [test_use_httpssvc.js]
 skip-if = os == "android"
+[test_trr_additional_section.js]
+skip-if = os == "android"
+
 [test_httpssvc_iphint.js]
 skip-if = os == "android"
